Modules/Devices/src/Devices.Application/Clients/Commands/UpdateClient/Handler.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Backbone.Modules.Devices.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Devices.Domain.Aggregates.Tier;
+using Enmeshed.BuildingBlocks.Application.Abstractions.Exceptions;
+using MediatR;
+using ApplicationException = Enmeshed.BuildingBlocks.Application.Abstractions.Exceptions.ApplicationException;
+
+namespace Backbone.Modules.Devices.Application.Clients.Commands.UpdateClient;
+public class Handler : IRequestHandler<UpdateClientCommand, UpdateClientResponse>
+{
+    private readonly IOAuthClientsRepository _oAuthClientsRepository;
+    private readonly ITiersRepository _tiersRepository;
+
+    public Handler(IOAuthClientsRepository oAuthClientsRepository, ITiersRepository tiersRepository)
+    {
+        _oAuthClientsRepository = oAuthClientsRepository;
+        _tiersRepository = tiersRepository;
+    }
+
+    public async Task<UpdateClientResponse> Handle(UpdateClientCommand request, CancellationToken cancellationToken)
+    {
+        var client = await _oAuthClientsRepository.Find(request.ClientId, cancellationToken);
```

Comment:
This repository uses the following for changes to clients:
`OpenIddictApplicationManager<CustomOpenIddictEntityFrameworkCoreApplication> _applicationManager`

I don't really think we can add tracking for this.

Reply:
True. But in order to stay consistent, we should still add the parameter, even though we cannot use it. Because the caller of the repository shouldn't know about the actual implementation. It may as well be that there is an implementation that DOES support tracking.